torch/csrc/jit/symbolic_variable.h

```diff
@@ -63,6 +63,42 @@ struct SymbolicVariable {
     n->t_(attr::other, rhs.toTensor());
     return r;
   }
+  SymbolicVariable operator>(at::Scalar rhs) const {
+    Node * n;
+    auto r = create(aten::gt, {*this}, 1, &n)[0].typeLike(*this);
+    n->t_(attr::other, rhs.toTensor());
+    return r;
+  }
+  SymbolicVariable operator<(at::Scalar rhs) const {
+    Node * n;
+    auto r = create(aten::lt, {*this}, 1, &n)[0].typeLike(*this);
```

Comment:
Hmmm should it really be `typeLike(*this)` and not `Byte`?

Reply:
Good catch, how do I set type to Byte?

Reply:
It looks like you'll have to implement it yourself. Internally `typeLike` is working with `Type` and then you can use `toScalarType` to get it over to the correct type.